.github/workflows/static_analysis.yml

```diff
@@ -27,3 +28,59 @@ jobs:
           distribution: 'zulu'
       - name: Lint
         run: ./gradlew lint
+
+  diffuse:
+    name: Diffuse AAR Analysis
+    runs-on: macOS-11
+    steps:
+      # Set up environment to assemble SDK
+      - name: Set up Java 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'zulu'
+
+      # Assemble artifacts from main branch
+      - name: Checkout Main Branch
+        uses: actions/checkout@v2
+        with:
+          ref: main
+      - name: Assemble Release AAR on Main Branch
+        run: ./gradlew clean assembleRelease
+      - name: Upload Main Branch Artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: main-aar
+          path: |
+            Core/build/outputs/aar/Core-release.aar
+            Card/build/outputs/aar/Card-release.aar
+
+      # Assemble artifacts from current branch
+      - name: Checkout Current Branch
+        uses: actions/checkout@v2
+      - name: Assemble Release AAR on Current Branch
+        run: ./gradlew clean assembleRelease
+      - name: Upload Current Branch Artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: current-aar
+          path: |
+            Core/build/outputs/aar/Core-release.aar
+            Card/build/outputs/aar/Card-release.aar
+
+      # Set up Diffuse
+      - name: Install Diffuse
+        run: brew install JakeWharton/repo/diffuse
+      - name: Download Workflow Artifacts 
+        uses: actions/download-artifact@v2
+      - name: Set up Java 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      # Run Diffuse analysis
+      - name: Run Diffuse Analysis - Card
+        run: diffuse diff --aar main-aar/Card/build/outputs/aar/Card-release.aar current-aar/Card/build/outputs/aar/Card-release.aar
+      - name: Run Diffuse Analysis - Core
```

Comment:
The run steps and the upload / download artifact steps would have to be modified for each new module added. With this approach they each get their own std output report printed in the action. Do we think this is an OK approach? 

Alternatively - we could compare demo app APK size  (if the demo app includes all modules any changes in module size would be reflected in the demo app APK size change right?). This would allow us to only run diffuse once, but would provide less granular insight into actual SDK size.

Reply:
I agree with @sshropshire. Having granular reports will help pinpoint which module needs to be investigated if we see a large increase in SDK size.